app/models/experience.rb

```diff
@@ -48,7 +48,8 @@ class Experience < ActiveRecord::Base
   validates :link,        url: true, allow_blank: true
 
   def title
-    self.description.present? ? self.description.truncate(50) : nil
+    # Since we allow RTF in the description, we should strip any HTML from the title
+    self.description.present? ? ActionController::Base.helpers.strip_tags(self.description).truncate(30) : nil
```

Comment:
It just looked a little long relative to the objects with _actual_ titles that are rendered in the same list (on user#show). Happy to bump back up if you prefer!